CHANGELOG.md

```diff
@@ -323,3 +323,17 @@
     - Fixed an issue where multiple store credit reversals occurred due to failed payment authorizations.
     - Resolved a race condition causing the Bolt Product Page button to occasionally fail to render.
 
+## [v2.27.6](https://github.com/BoltApp/bolt-magento2/releases/tag/2.27.6) 2025-02-28
+- New:
+    - Added support for Magento versions 2.4.6-p9 and 2.4.7-p4, ensuring compatibility with the latest Magento security patch. Specifically, the ACL list for the Bolt API has been updated.
+- Additional:
+    - Fixed an issue where disabling address pre-filling for logged-in customers was not working as expected.
+    - Removed Bolt inline script usage on the frontend, which could cause bugs with certain environment configurations.
+    - Fixed merchant Bolt dashboard information when BO orders were placed using customers' saved cards.
```

Comment:
I think this is Bolt merchant dashboard, since the product is called "merchant dashboard" not "Bolt dashboard".